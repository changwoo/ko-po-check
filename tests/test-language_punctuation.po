# -*-text-*-
#
msgid ""
msgstr ""
"Project-Id-Version: test 13\n"
"POT-Creation-Date: 2012-05-29 22:19+0900\n"
"PO-Revision-Date: 2012-05-29 22:19+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: KO-PO-CHECK <cwryu@debian.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# 세미콜론
msgid "Blahblah blahblah; Blahblah."
msgstr "어쩌구 어쩌구입니다; 어쩌구입니다."

msgid "Blahblah blahblah; Blahblah."
msgstr "어쩌구 blah어쩌구입니다; 어쩌구입니다."

msgid "Blahblah blahblah;\n"
"%s"
msgstr "어쩌구 blah어쩌구입니다;\n"
"%s"

# 세미콜론 예외, desktop keywords
#: ../data/test.desktop.in.in.h:1
msgid "Apple;Banana;"
msgstr "사과;바나나;"

# 세미콜론 예외, from gettext-tools
msgid "message catalog has plural form translations, but lacks a header entry with \"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\""
msgstr "메시지 목록에 복수형 번역이 들어 있지만, 헤더 항목에 \"Plural-Forms: nplurals=정수; plural=표현식;\"이 없습니다"
