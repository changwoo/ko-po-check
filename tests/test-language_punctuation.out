-:16: 맞춤법 규정에 따라 괄호 앞에 띄어 쓰지 않습니다
